Formula/tomsfastmath.rb

```diff
@@ -0,0 +1,64 @@
+class Tomsfastmath < Formula
+  desc "Fast large integer arithmetic library written in portable ISO C"
+  homepage "https://www.libtom.net/TomsFastMath/"
+  url "https://github.com/libtom/tomsfastmath/releases/download/v0.13.1/tfm-0.13.1.tar.xz"
+  sha256 "47c97a1ada3ccc9fcbd2a8a922d5859a84b4ba53778c84c1d509c1a955ac1738"
```

Comment:
I'm aware that there should be a license here, but I'm not sure what to put. The current development branch is under the Unlicense, but the latest release (0.13.1) is dual-licensed under some custom public domain dedication and the WTFPL. See https://github.com/libtom/tomsfastmath/blob/v0.13.1/LICENSE.

What should I do?

Reply:
Could you ask for a release?

Reply:
Done: https://github.com/libtom/tomsfastmath/issues/29

Reply:
My issue hasn't received any response, so I think we can put the dual license of the 0.13.1 release here?

Reply:
Done. Is `:public_domain` appropriate for a custom public domain dedication?